install.sh

```diff
@@ -115,6 +115,16 @@ rosdep -q update
 echo -e "\e[32mDone: Configuring rosdep\e[0m"
 echo ""
 
+
+echo -e "\e[94mSetting up groups\e[0m"
+sudo addgroup flirimaging
+sudo usermod -a -G flirimaging administrator
+sudo sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT="[^"]*/& usbcore.usbfs_memory_mb=1000/' /etc/default/grub
```

Comment:
Since some computers don't have the grub file, should we check for that and also make sure that we don't add this same text twice if users run the script again? (e.g. if later steps fail and they rerun it to finish up setup)

Reply:
Updated.